evm/upgradable.js

```diff
@@ -42,6 +42,26 @@ async function deployCreateUpgradable(
     return new Contract(proxy.address, implementationJson.abi, wallet);
 }
 
+async function deployCreate2(
+    constAddressDeployerAddress,
+    wallet,
+    contractJson,
+    key = Date.now(),
+    args = [],
+    gasOptions = null,
+    env = 'testnet',
+    chain = 'ethereum',
+    verify = false,
+) {
+    const implementation = await deployContractConstant(constAddressDeployerAddress, wallet, contractJson, key, args, gasOptions?.gasLimit);
```

Comment:
there is no notion of implementation here
```suggestion
    const contract = await deployContractConstant(constAddressDeployerAddress, wallet, contractJson, key, args, gasOptions?.gasLimit);
```